src/index.js

```diff
@@ -37,19 +37,21 @@ export default class SHA256 {
   }
 
   static digest(data) {
+    if (data.length == 64) {
```

Comment:
Use triple equals `===`. If there's a strong reason to use `==` please add a comment